core/src/main/java/edu/wpi/grip/core/operations/networktables/NTPublishOperation.java

```diff
@@ -28,9 +28,12 @@
     private final List<Method> ntValueMethods = new ArrayList<>();
 
     public NTPublishOperation(Class<T> type) {
-        this.table = NetworkTable.getTable("GRIP");
         this.type = checkNotNull(type, "Type was null");
 
+        // NetworkTables automatically initializes itself when we get a table.  We don't want this behavior.
+        table = NetworkTable.getTable("GRIP");
+        NetworkTable.shutdown();
```

Comment:
Yes.  When you load a project, a `ProjectSettingsChangedEvent` is sent, and `NTManager` hears this and calls `NetworkTables::initialize()`.